src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Models/CmdletParamEnums.cs

```diff
@@ -30,4 +30,9 @@ public enum ContainerParams
         ResourceGroupName,
         Status,
     }
+
+    public enum CommonParams
+    {
+        BackupManagementType,
```

Comment:
fixed. When i coded policyparams was not thr so created a common params so that every one can use it.